src/credentials/CHIPCert.cpp

```diff
@@ -677,15 +681,15 @@ CHIP_ERROR ChipDN::GetCertFabricId(uint64_t & fabricId) const
 {
     uint8_t rdnCount = RDNCount();
 
-    fabricId = UINT64_MAX;
+    fabricId = kUndefinedFabricIndex;
```

Comment:
I don't see how that makes sense.  The LHS is not a fabric _index_.

Reply:
Of course! My typo. Fixed that